operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/client/BulkDataClient.java

```diff
@@ -734,6 +739,10 @@ private PollingLocationResponse process(JobExecutionResponse response) {
                 }
             }
             result.setOutput(outputList);
+
+            // Errors need to be added.
+            List<PollingLocationResponse.Output> errors = Collections.emptyList();
+            result.setError(errors);
```

Comment:
do we ever set a non-empty list?  i thought we would put OperationOutcome for an resources that failed to export into this list.

Reply:
We don't.  The only time we use it is when we do the Bulk Import.  We have a backlog item  Add operationoutcomes for errors during bulkdata export #1161